.github/workflows/packaging.yml

```diff
@@ -302,8 +302,14 @@ jobs:
       - name: Extract source archive
         run: |
           tar xf apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz
+
+          source ./apache-arrow-adbc-${{ steps.info.outputs.VERSION }}/dev/release/versions.env
+
           mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }} adbc
-          mv apache-arrow-adbc-${{ steps.info.outputs.VERSION }}.tar.gz adbc/ci/linux-packages/
+          # Need to align the file path with the version number (and not the release)
```

Comment:
Ah, we use `apache-arrow-adbc-YYYY.MM` tag after this PR?
(`VERSION` is `YYYY.MM` and `VERSION_NATIVE` is `1.0.0`, right?)

Reply:
Yup. Though I'm undecided if it should be MM or just a counter.

Reply:
ah, well, if we add the day below then it will be a date and that sounds good to me.